core/src/navigation/ProductSwitcher.html

```diff
@@ -0,0 +1,83 @@
+<div class="fd-shellbar__action fd-shellbar__action--collapsible">
```

Comment:
When clicking any of the project links, the popover does not disappear. Please make it disappear to be consistent with context switcher.